docs/api/security/trusted_storage.md

```diff
@@ -0,0 +1,32 @@
+<h2 id="blockdevice-port">PSA Internal Trusted Storage Reference API</h2>
+
+## PSA Internal Trusted Storage
+
+The PSA Internal Trusted Storage is a PSA Root of Trust Service which provides key (uid) value secure storage in internal flash for small data items.
+The main use cases for PSA Internal Trusted Storage APIs are:
+- Provide storage solution for persistent keys in PSA Crypto Service.
+- Store rollback protection tokens for external application secure storage.
```

Comment:
Should we call this "replay protection" from now on?

Reply:
@geky, i tend to disagree. It is a first time i encounter "replay protection" terminology, while "rollback protection" is widely adopted term in security domain.

google both term by yourself and see the difference.

@dreemkiller - you are the security expert here - please vote.

Reply:
This came from a discussion with @dreemkiller and @flit. From what I understand we're NOT providing strict rollback protection as it's known in security. So we are prefering "replay protection" to avoid bad assumptions.

But I'm not an expert though, just didn't know if it was overlooked. So don't let me block anything :+1: